apps/frontend/src/app/Components/Character/CharacterCard.tsx

```diff
@@ -379,43 +373,43 @@ function HeaderContent() {
         color={characterEle}
         sx={{ opacity: 0.85 }}
       />
-      <Grid container spacing={1} flexWrap="nowrap">
-        <Grid item sx={{ textShadow: '0 0 5px gray' }}>
+      <Box display="flex" gap={1} sx={{ textShadow: '0 0 5px gray' }}>
+        <Box>
           <Typography component="span" variant="h6" whiteSpace="nowrap">
             Lv. {characterLevel}
           </Typography>
           <Typography component="span" variant="h6" color="text.secondary">
             /{ascensionMaxLevel[ascension]}
           </Typography>
-        </Grid>
-        <Grid item>
-          <Typography variant="h6">
-            <SqBadge>C{constellation}</SqBadge>
-          </Typography>
-        </Grid>
-      </Grid>
-      <Grid container spacing={1} flexWrap="nowrap">
-        <Grid item>
-          <Chip
-            color={autoBoost ? 'info' : 'secondary'}
-            label={<strong>{tAuto}</strong>}
-          />
-        </Grid>
-        <Grid item>
-          <Chip
-            color={skillBoost ? 'info' : 'secondary'}
-            label={<strong>{tSkill}</strong>}
-          />
-        </Grid>
-        <Grid item>
-          <Chip
-            color={burstBoost ? 'info' : 'secondary'}
-            label={<strong>{tBurst}</strong>}
-          />
-        </Grid>
-      </Grid>
-      <Typography mt={1}>
-        <StarsDisplay stars={characterSheet.rarity} colored />
+        </Box>
+        <Typography
```

Comment:
Not a fan of this. maybe make the sqbadge the color? But having the text as the color looks tacky and makes it hard to read

Reply:
Sqbadge always looked weird because of the padding, would need to like half the font size and flex box vertical align, which probably looks weird...
<img width="220" alt="image" src="https://user-images.githubusercontent.com/1754901/231233045-c4c13627-7ff7-481c-a51e-2b2e85670bdb.png">

Reply:
Looks way better imo